ui/app/store/actions.js

```diff
@@ -2487,8 +2487,16 @@ export function setAlertEnabledness(alertId, enabledness) {
   }
 }
 
-export async function setUnconnectedAccountAlertShown(origin) {
-  await promisifiedBackground.setUnconnectedAccountAlertShown(origin)
+export function setUnconnectedAccountAlertShown(origin) {
+  return async () => {
```

Comment:
[Q] Do these have to be thunks? There was conversation in a recent PR (that I'll try and bring up) about not using thunks/dispatch when you don't need to access dispatch/state in the thunk.

Reply:
https://github.com/MetaMask/metamask-extension/pull/9892#discussion_r524652940

Reply:
For the sake of consistency, I un-thunkified all of the alert-related ones: 4cb4bfa